scripts/e2e.sh

```diff
@@ -21,7 +21,7 @@ KUBERNETES_VERSION=v1.8.0
 KUBE_STATE_METRICS_LOG_DIR=./log
 KUBE_STATE_METRICS_IMAGE_NAME='quay.io/coreos/kube-state-metrics'
 KUBE_STATE_METRICS_IMAGE_NAME_PATTERN='quay.io\/coreos\/kube-state-metrics'
-PROMETHEUS_VERSION=1.8.1
+PROMETHEUS_VERSION=2.0.0
```

Comment:
With prometheus version bump to 2.0.0, the check metrics command for promtool should be `promtool check-metrics` instead of `promtool check metrics`.